src/Microsoft.VisualStudio.Web.CodeGeneration.Core/Microsoft.VisualStudio.Web.CodeGeneration.Core.csproj

```diff
@@ -0,0 +1,45 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\dir.props" />
+  <PropertyGroup>
+    <Description>Contains the core infrastructure used by ASP.NET Core Code Generators.</Description>
+    <TargetFrameworks>net451;netstandard1.6</TargetFrameworks>
+    <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <AssemblyName>Microsoft.VisualStudio.Web.CodeGeneration.Core</AssemblyName>
+    <PackageTags>aspnetcore;codegenerator;scaffolding;visualstudioweb</PackageTags>
+    <PackageTargetFallback Condition=" '$(TargetFramework)' == 'netstandard1.6' ">$(PackageTargetFallback);portable-net45+win8</PackageTargetFallback>
+    <PackageVersion>$(VersionPrefix)-$(VersionSuffix)</PackageVersion>
```

Comment:
This is actually something else. 
Since these projects depend each other using `<ProjectReference ...>` Not having this here, causes the generated package to depend on version `1.0.0` of the dependency.